examples/point-lighting.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+        <meta
+            name="viewport"
+            content="width=device-width, minimal-ui, viewport-fit=cover, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0, user-scalable=no"
+        />
+        <link rel="icon" type="image/png" href="assets/favicon.png" />
+
+        <title>OGL • Point lighting with specular highlights</title>
+        <link href="assets/main.css" rel="stylesheet" />
+        <style>
+            body {
+                color: white;
+            }
+        </style>
+    </head>
+    <body>
+        <div class="Info">
+            Point lighting with specular highlights. Texture by
+            <a href="https://www.solarsystemscope.com/textures/" target="_blank">Solar System Scope</a>
+        </div>
+        <script type="module">
+            import { Renderer, Camera, Vec3, Orbit, Sphere, Transform, Geometry, Program, Mesh, Texture } from '../src/index.mjs';
+
+            const VERTEX_SHADER = `
+            attribute vec2 uv;
+            attribute vec3 position;
+            attribute vec3 normal;
+
+            uniform mat4 modelMatrix;
+            uniform mat4 modelViewMatrix;
+            uniform mat4 projectionMatrix;
+
+            uniform vec3 u_lightWorldPosition;
+            uniform vec3 cameraPosition;
+            
+
+            varying vec3 v_normal;
+            varying vec3 v_surfaceToLight;
+            varying vec3 v_surfaceToView;
+            varying vec2 v_uv;
+
+            void main() {
+                // Pass UV information to Fragment Shader
+                v_uv = uv;
+
+                // Calculate World Space Normal
+                v_normal = mat3(modelMatrix) * normal;
```

Comment:
why not normalMatrix ?

Reply:
Hello, I don't fully understand the nuances between using the modelMatrix and normalMatrix in this case. While testing it out both seemed to produce the same result.

Reply:
In short normalMatrix will work even in cases where you have non uniform scale of mesh

Reply:
I see. Just so I have it right, i should multiply normalMatrix with the normal, just like the modelMatrix correct?

Reply:
Yes and its mat3 already

Reply:
Cool, will make that change. Appreciate the feedback.

Reply:
In case you would like to know more about basic light shaders, this [book](https://www.apress.com/gp/book/9781484244562) have very good chapter on them :)

Reply:
That helps, thanks!